README.md

```diff
@@ -74,15 +73,14 @@ bin/vast --help
 To install VAST properly for your local user simly place the unpacked folders in
 `/usr/local/`.
 
-MacOS and FreeBSD users have to build from source. Clone the `master` branch to
+macOS and FreeBSD users have to build from source. Clone the `master` branch to
```

Comment:
I wouldn't start a sentence with lowercase. Here, you can void this elegantly by starting the sentence with "FreeBSD".

Reply:
Sounds like a good idea, will swap the order.